apps/aechannel/src/aesc_fsm.erl

```diff
@@ -714,13 +692,8 @@ accepted(cast, {?FND_CREATED, Msg}, #data{role = responder} = D) ->
         %% {error, _} = Error ->
         %%     close(Error, D)
     end;
-accepted(cast, {?DISCONNECT, _Msg}, D) ->
-    close(disconnect, D);
-accepted(timeout, accepted = T, D) ->
-    close({timeout, T}, D);
-accepted({call, From}, Req, D) ->
-    handle_call(accepted, Req, From, D).
-
+accepted(Type, Msg, D) ->
+    handle_common_event(Type, Msg, error, D).
```

Comment:
What is the difference between `error` and `error_all`?

Reply:
What I got is `error_all` has some special meaning (still opening)

Reply:
In the `error` mode, calls are still handled (but could potentially be postponed by `handle_call()` code). In `error_all` all calls and casts not explicitly handled lead to protocol error. This is used during the initial start/reestablish handshake.